lua/lspconfig/server_configurations/reason_ls.lua

```diff
@@ -0,0 +1,25 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = {},
```

Comment:
Can we just put the binary name here anyways? That way people can add it to their path.

Reply:
Yes, I have been add the binary name in `cmd`.